### PR TITLE
build scripts for multiple java packages

### PR DESCRIPTION
**Pull request by Sapana-Khemkar**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: c/commons-fileupload/commons-fileupload_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# ---------------------------------------------------------------------------
```
Comment (amitsadaphule) [inline c/commons-fileupload/commons-fileupload_ubi_8.4.sh:1]: LICENSE file is missing in the directory. Please add it.

File: g/google-api-client-jackson2/google-api-client-jackson2_ubi8.4.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# ----------------------------------------------------------------------------
+#
+# Package	: google-api-client-jackson2
+# Version	: 1.27.0
+# Source repo	: https://github.com/googleapis/google-api-java-client/tree/v1.27.0
```
Comment (amitsadaphule) [inline g/google-api-client-jackson2/google-api-client-jackson2_ubi8.4.sh:5]: Please correct the repo url.

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: g/google-api-client-jackson2/google-api-client-jackson2_ubi8.4.sh (hunk 2 above)
Comment (Sapana-Khemkar) [reply g/google-api-client-jackson2/google-api-client-jackson2_ubi8.4.sh]: Corrected.

File: c/commons-fileupload/commons-fileupload_ubi_8.4.sh (hunk 1 above)
Comment (Sapana-Khemkar) [reply c/commons-fileupload/commons-fileupload_ubi_8.4.sh:1]: Done

**Review by amitsadaphule — APPROVED** [review]
LGTM

**Comment by amitsadaphule** [comment]
@seth-priya not sure what should be the commit message here, deferring merge (Too many packages) :)

**Comment by seth-priya** [comment]
@Sapana-Khemkar @amitsadaphule  - maybe we can provide a comma separated list of all package names if that can fit

**Pull request merged**